pom.xml

```diff
@@ -153,6 +153,15 @@
 		  <scope>test</scope>
 	  </dependency>
 
+	  <!-- https://mvnrepository.com/artifact/org.springframework.security/spring-security-crypto -->
```

Comment:
Je opravdu potreba to mit na hlavni urovni? Pouzijes to ve vice modulech nebo jen v jednom?